src/index.html

```diff
@@ -6,8 +6,276 @@
+          <div class="testimonials__cards">
+            <div class="testimonial">
+              <img
+                src="./images/people/1.jpg"
+                alt="photo of Leslie Mckinney"
+                class="testimonial__portret"
+              >
+              <h3 class="testimonial__name page-title-3">Leslie Mckinney</h3>
+              <p class="testimonial__position page-gray-text">
+                <span class="page-gray-text">CEO</span>
+              </p>
+              <p class="testimonial__description page-text">
+                Nice work, Certificates National Association of Realtors (USA)
+              </p>
+            </div>
+            <div class="testimonial">
+              <img
+                src="./images/people/2.jpg"
+                alt="photo of Scarlett Hawkins"
+                class="testimonial__portret"
+              >
+              <h3 class="testimonial__name page-title-3">Scarlett Hawkins</h3>
+              <p class="testimonial__position">
+                <span class="page-gray-text">Senior Vice President</span>
+              </p>
+              <p class="testimonial__description page-text">
+                My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+              </p>
+            </div>
+            <div class="testimonial">
+              <img
+                src="./images/people/3.jpg"
+                alt="photo of Max Henry"
+                class="testimonial__portret"
+              >
+              <h3 class="testimonial__name page-title-3">Max Henry</h3>
+              <p class="testimonial__position">
+                <span class="page-gray-text">Chief Design Officer</span>
+              </p>
+              <p class="testimonial__description page-text">
+                When my lovely valley of steam rises and half-day sun is above an impermeable
+              </p>
+            </div>
+          </div>
+        </section>
+        <section class="contacts" id="contacts">
+          <div class="contacts__title">
+            <h2 class="page-title-2">Contact us</h2>
+            <div class="title-decoration"></div>
+          </div>
+          <div class="contacts__content">
+            <div class="contacts__e-contacts">
+              <div class="phone">
+                <p class="page-gray-text">Phone</p>
+                <p class="page-text">+1 234 5555-55-55</p>
+              </div>
+              <div class="email">
+                <p class="page-gray-text">Email</p>
+                <p class="page-text">hello@miami.com</p>
+              </div>
+            </div>
+            <div class="contacts__address">
+              <p class="page-gray-text">Address</p>
+              <p class="page-text">
+                400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401
+              </p>
+            </div>
+            <form
+              action="./index.html"
+              method="post"
```

Comment:
```suggestion
              action="#"
              method="GET"
```
It may fix problem with error